src/openrct2/actions/CommandFlag.h

```diff
@@ -0,0 +1,44 @@
+/*****************************************************************************
+ * Copyright (c) 2014-2025 OpenRCT2 developers
+ *
+ * For a complete list of all authors, please refer to contributors.md
+ * Interested in contributing? Visit https://github.com/OpenRCT2/OpenRCT2
+ *
+ * OpenRCT2 is licensed under the GNU General Public License version 3.
+ *****************************************************************************/
+
+#pragma once
+
+#include "../core/FlagHolder.hpp"
+
+#include <cstdint>
+
+namespace OpenRCT2::GameActions
+{
+    enum class CommandFlag : uint8_t
+    {
+        /**
+         * If this flag is set, the command is applied, otherwise only the cost is retrieved.
+         */
+        apply,
+        /**
+         * Command was issued from replay manager.
+         */
+        replay,
+        allowDuringPaused = 3,
+        /**
+         * Game command is not networked
+         */
+        noSpend = 5,
+        /**
+         * Game command is not networked
+         */
+        ghost = 6,
+        trackDesign = 7,
+        /**
+         * Game command is coming from network
+         */
```

Comment:
Any particular reason for going with doxygen comment style/syntax here? This doesn't seem to be common in our codebase. I'd prefer to stay with single-line and/or trailing comments in these instances, personally.